tests/release/e2e-test-releaseplan-owner-ref.go

```diff
@@ -51,7 +52,7 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-2469]test-releaseplan-owner-ref-a
 			Eventually(func() error {
 				releasePlan, err = fw.AsKubeAdmin.ReleaseController.GetReleasePlan(sourceReleasePlanName, devNamespace)
 				if !errors.IsNotFound(err) {
-					return fmt.Errorf("ReleasePlan %s for application %s still not deleted\n", releasePlan.GetName(), applicationNameDefault)
+					return fmt.Errorf("ReleasePlan %s for application %s still not deleted", releasePlan.GetName(), applicationNameDefault)
```

Comment:
Unrelated change.

Reply:
I'm not saying that it doesn't have to be fixed. I'm saying that it should not be part of this change as it is unrelated.

Reply:
I agree with David in general on this, but for such a cosmetic change, I would be totally fine to just fix it here instead of requiring a separate PR. Just my $0.02 :)